README.md

```diff
@@ -585,12 +586,14 @@ And finally, let's cherry-pick the commit for bug #14:
 
 <a name="delete-stale-local-branches"></a>
 ### I want to delete local branches that were deleted upstream
-Once you merge a pull request on github, it gives you the option to delete the merged branch in your fork. If you aren't planning to keep working on the branch, it's cleaner to delete the local copies of the branch so you don't end up cluttering up your working checkout with a lot of stale branches.
+Once you merge a pull request on GitHub, it gives you the option to delete the merged branch in your fork. If you aren't planning to keep working on the branch, it's cleaner to delete the local copies of the branch so you don't end up cluttering up your working checkout with a lot of stale branches.
 
 ```sh
-$ git fetch -p
+$ git fetch -p upstream
 ```
 
+where, `upstream` is the e remote you want to fetch from.
```

Comment:
Spelling error?

Reply:
Yep.

Reply:
@sivaraam No need to comment - just amend the commit or add a new one and my comment will go away. :)